src/styles/index.scss

```diff
@@ -1,3 +1,82 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  position: relative;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__minute-hand {
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -100%);
+    transform-origin: bottom;
+    z-index: -1;
+    animation: move-minute-hand 3600s steps(60) infinite;
```

Comment:
The `z-index: -1;` on line 33 can cause the `.stopwatch__minute-hand` to appear below other positioned elements that have a default stacking order (z-index of 0). Make sure this is the intended behavior; otherwise, consider using a `z-index` value greater than or equal to 0.